modules/ROOT/pages/telemetry-exporter.adoc

```diff
@@ -29,15 +29,18 @@ Azure Monitor and Splunk HEC are also compatible.
 
 Before you configure the Telemetry Exporter, ensure that you have:
 
-//include::reuse::partial$billing/pricing.adoc[tag=prereqBullet] - Removing for now. Including as plain text. 
-//retain refs to Advanced and Titanium
-* The Anypoint Integration Advanced package or a Titanium subscription to Anypoint Platform. 
-* Telemetry Exporter Administrator permission, so you can create or modify connections. For more information, see xref:am-permissions.adoc[].
-* Telemetry Exporter Configurations Manager permission, so you can create or modify configurations. For more information, see xref:am-permissions.adoc[].
-* Access information for your third-party observability platform.
+* Permissions that you require for your uses of Telemetry Exporter (see xref:am-permissions.adoc[])
+** *Telemetry Exporter Administrator* permission for creating and modifying connections
+** *Telemetry Exporter Configurations Manager* permission for creating and modifying configurations
+* Access information for your third-party observability platform
 * The ability to create an instrumentation key or access token for your third-party observability platform.
-* Enabled trace data collection for CloudHub 2.0 apps deployed to xref:cloudhub-2::ch2-deploy-shared-space.adoc#enable-trace-data-collection[shared] or xref:cloudhub-2::ch2-deploy-private-space.adoc#enable-trace-data-collection[private spaces]. This requirement applies only if you want to export trace data for CloudHub 2.0.
-* Enabled trace data collection for xref:runtime-fabric::use-anypoint-monitoring.adoc#enable-trace-data[Runtime Fabric] apps. This requirement applies only if you want to export trace data for Runtime Fabric.
+* Enabled trace data collection for your Mule apps:
+** For CloudHub 2.0 deployments, see xref:cloudhub-2::ch2-deploy-shared-space.adoc#enable-trace-data-collection[shared] and xref:cloudhub-2::ch2-deploy-private-space.adoc#enable-trace-data-collection[private spaces].
+** For Runtime Fabric deployments, see xref:runtime-fabric::use-anypoint-monitoring.adoc#enable-trace-data[Runtime Fabric].
```

Comment:
NEW CONTENT for this issue below:

Reply:
@jigarkb can you take a look at the new content below. The content above is just reorganized and unrelated to this issue.